### ISSUE TITLE
Inventory Rework: Folder Descriptions and Item Descriptors

### ISSUE THREAD

**Issue by TehTurk** [Neos-Metaverse/NeosPublic#1975]
## Is your feature request related to a problem? Please describe.
Information about a folder or a item is problematic, something to alleviate this would be ideal.

## Describe the solution you'd like
I'd be ideal if we had some descriptions for folders in some way, not through tool hints but something tied in the UI itself so people can know things like "Owner, Creator, etc," or metadata involved with the object.

## Additional context
- brought up by BusinessLawyer and dfghiatus

**Comment by Frooxius** [comment]
I don't understand this issue, what exactly is the problem you're trying to solve? What kind of information is missing? Naming of the inventory items and folders is fully controlled by the user.

**Comment by TehTurk** [comment]
This an issue for ideas and considerations for the new inventory UI once it's reworked. Say if you focus a Folder, you need to know maybe a description or there needs to be more context then just a name for it. Even something like a context description.

 In the case of an item, you'd need things like original owner, date made, etc just general metadata or information regarding said item.

**Comment by Frooxius** [comment]
This is planned to be added. You'll be able to view various metadata in the reworked UI. Internally the data is already there.

**Comment by TehTurk** [comment]
Yup! Noting this down as an issue for the future, so people have an issue to refer to, but also so it gives a clear idea of what's been suggested and confirm what's going to be added or not.
